includes/discovery/sensors/load/rfc1628.inc.php

```diff
@@ -7,7 +7,7 @@
 foreach ($load_data as $index => $data) {
     $load_oid = ".1.3.6.1.2.1.33.1.4.4.1.5.$index";
 
-    if (is_array($data['upsOutputPercentLoad'])) {
+    if (isset($data['upsOutputPercentLoad'][0])) {
```

Comment:
Any reason this was changed?

Reply:
Yes, @murrant suggested it was a better way to check, so I updated all references to the upsDataFoo from is_array to isset.